src/tyr/route_serializer_osrm.cc

```diff
@@ -1494,65 +1503,94 @@ json::ArrayPtr voice_instructions(const valhalla::DirectionsLeg::Maneuver* prev_
   // shortly (10 meters at the given speed) after the maneuver has started.
   // The voice_instruction_beginning starts shortly after the beginning of the step.
   // The voice_instruction_end starts shortly before the end of the step.
-  float distance_before_verbal_transition_alert_instruction = -1;
-  float distance_before_verbal_pre_transition_instruction = -1;
-  if (prev_maneuver) {
```

Comment:
Just moved this `if(prev_maneuver)` out of the function. We should always have it.